api/v1beta1/workspace_types.go

```diff
@@ -162,6 +162,8 @@ type TuningSpec struct {
 	Input *DataSource `json:"input"`
 	// Output specified where to store the tuning output.
 	Output *DataDestination `json:"output"`
+	// TtlSecondsAfterFinished specifies the time to live for the tuning workspace after it has finished.
+	TtlSecondsAfterFinished *int64 `json:"ttlSecondsAfterFinished,omitempty"`
```

Comment:
how about defining this field with reference to Job.Spec.TTLSecondsAfterFinished filed?

```
TtlSecondsAfterFinished *int64 --> TTLSecondsAfterFinished *int32
```